src/software/thunderscope/gl/widgets/gl_shift_toolbar.py

```diff
@@ -0,0 +1,50 @@
+from pyqtgraph.Qt import QtGui
+from pyqtgraph.Qt.QtCore import pyqtSignal
+from pyqtgraph.Qt.QtWidgets import *
+from proto.import_all_protos import *
+from software.thunderscope.gl.widgets.gl_toolbar import GLToolbar
+
+
+class ShiftButtonToolbar(GLToolbar):
```

Comment:
I find this UI very confusing. Rather than a drop-down menu, it might be better as a push button where pushing the button in enables shift-click and the depressed button disables shift-click.

Also, I wonder if there are other ideas on this that @williamckha or @sauravbanna can suggest